views/sign-out.html

```diff
@@ -21,7 +21,7 @@
 
 {% if saveAndResume and (journey == "register") %}
   {% set hintText = "We will save your application." %}
-{% elif saveAndResume and (journey == "update") %}
+{% elif saveAndResume and (journey == "update" or journey == "remove") %}
```

Comment:
Is it maybe worth defining an `isUpdateOrRemoveJourney` variable at the top of this file to shorten this repeating logic check in the four times it's done? Or could you just do `journey != "register"` ? Of course, logic will further simplify if/when the non 'save and resume' code is ever ripped out...